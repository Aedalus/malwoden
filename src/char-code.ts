/// Unicode code points for various special characters that also exist on
/// [code page 437][font].
///
/// [font]: http://en.wikipedia.org/wiki/Code_page_437
export enum CharCode {
  // 1 - 15.
  whiteSmilingFace = 0x263a,
  blackSmilingFace = 0x263b,
  blackHeartSuit = 0x2665,
  blackDiamondSuit = 0x2666,
  blackClubSuit = 0x2663,
  blackSpadeSuit = 0x2660,
  bullet = 0x2022,
  inverseBullet = 0x25d8,
  whiteCircle = 0x25cb,
  inverseWhiteCircle = 0x25d9,
  maleSign = 0x2642,
  femaleSign = 0x2640,
  eighthNote = 0x266a,
  beamedEighthNotes = 0x266b,
  whiteSunWithRays = 0x263c,

  // 16 - 31.
  blackRightPointingPointer = 0x25ba,
  blackLeftPointingPointer = 0x25c4,
  upDownArrow = 0x2195,
  doubleExclamationMark = 0x203c,
  pilcrow = 0x00b6,
  sectionSign = 0x00a7,
  blackRectangle = 0x25ac,
  upDownArrowWithBase = 0x21a8,
  upwardsArrow = 0x2191,
  downwardsArrow = 0x2193,
  rightwardsArrow = 0x2192,
  leftwardsArrow = 0x2190,
  rightAngle = 0x221f,
  leftRightArrow = 0x2194,
  blackUpPointingTriangle = 0x25b2,
  blackDownPointingTriangle = 0x25bc,

  // 32 - 47.
  space = 0x0020,
  exclamationPoint = 0x0021,
  doubleQuote = 0x0022,
  numberSign = 0x0023,
  dollarSign = 0x0024,
  percent = 0x0025,
  ampersand = 0x0026,
  apostrophe = 0x0027,
  leftParenthesis = 0x0028,
  rightParenthesis = 0x0029,
  asterisk = 0x002a,
  plus = 0x002b,
  comma = 0x002c,
  minus = 0x002d,
  period = 0x002e,
  slash = 0x002f,

  // 48 - 63.
  zero = 0x0030,
  one = 0x0031,
  two = 0x0032,
  three = 0x0033,
  four = 0x0034,
  five = 0x0035,
  six = 0x0036,
  seven = 0x0037,
  eight = 0x0038,
  nine = 0x0039,
  colon = 0x003a,
  semicolon = 0x003b,
  lessThan = 0x003c,
  equals = 0x003d,
  greaterThan = 0x003e,
  questionMark = 0x003f,

  // 64 - 95.
  at = 0x0040,
  aUpper = 0x0041,
  bUpper = 0x0042,
  cUpper = 0x0043,
  dUpper = 0x0044,
  eUpper = 0x0045,
  fUpper = 0x0046,
  gUpper = 0x0047,
  hUpper = 0x0048,
  iUpper = 0x0049,
  jUpper = 0x004a,
  kUpper = 0x004b,
  lUpper = 0x004c,
  mUpper = 0x004d,
  nUpper = 0x004e,
  oUpper = 0x004f,
  pUpper = 0x0050,
  qUpper = 0x0051,
  rUpper = 0x0052,
  sUpper = 0x0053,
  tUpper = 0x0054,
  uUpper = 0x0055,
  vUpper = 0x0056,
  wUpper = 0x0057,
  xUpper = 0x0058,
  yUpper = 0x0059,
  zUpper = 0x005a,
  leftBracket = 0x005b,
  backSlash = 0x005c,
  rightBracket = 0x005d,
  caret = 0x005e,
  underscore = 0x005f,

  // 96 - 127.
  accent = 0x0060,
  aLower = 0x0061,
  bLower = 0x0062,
  cLower = 0x0063,
  dLower = 0x0064,
  eLower = 0x0065,
  fLower = 0x0066,
  gLower = 0x0067,
  hLower = 0x0068,
  iLower = 0x0069,
  jLower = 0x006a,
  kLower = 0x006b,
  lLower = 0x006c,
  mLower = 0x006d,
  nLower = 0x006e,
  oLower = 0x006f,
  pLower = 0x0070,
  qLower = 0x0071,
  rLower = 0x0072,
  sLower = 0x0073,
  tLower = 0x0074,
  uLower = 0x0075,
  vLower = 0x0076,
  wLower = 0x0077,
  xLower = 0x0078,
  yLower = 0x0079,
  zLower = 0x007a,
  leftBrace = 0x007b,
  pipe = 0x007c,
  rightBrace = 0x007d,
  tilde = 0x007e,
  house = 0x2302,

  // 128 - 143.
  latinCapitalLetterCWithCedilla = 0x00c7,
  latinSmallLetterUWithDiaeresis = 0x00fc,
  latinSmallLetterEWithAcute = 0x00e9,
  latinSmallLetterAWithCircumflex = 0x00e2,
  latinSmallLetterAWithDiaeresis = 0x00e4,
  latinSmallLetterAWithGrave = 0x00e0,
  latinSmallLetterAWithRingAbove = 0x00e5,
  latinSmallLetterCWithCedilla = 0x00e7,
  latinSmallLetterEWithCircumflex = 0x00ea,
  latinSmallLetterEWithDiaeresis = 0x00eb,
  latinSmallLetterEWithGrave = 0x00e8,
  latinSmallLetterIWithDiaeresis = 0x00ef,
  latinSmallLetterIWithCircumflex = 0x00ee,
  latinSmallLetterIWithGrave = 0x00ec,
  latinCapitalLetterAWithDiaeresis = 0x00c4,
  latinCapitalLetterAWithRingAbove = 0x00c5,

  // 144 - 159.
  latinCapitalLetterEWithAcute = 0x00c9,
  latinSmallLetterAe = 0x00e6,
  latinCapitalLetterAe = 0x00c6,
  latinSmallLetterOWithCircumflex = 0x00f4,
  latinSmallLetterOWithDiaeresis = 0x00f6,
  latinSmallLetterOWithGrave = 0x00f2,
  latinSmallLetterUWithCircumflex = 0x00fb,
  latinSmallLetterUWithGrave = 0x00f9,
  latinSmallLetterYWithDiaeresis = 0x00ff,
  latinCapitalLetterOWithDiaeresis = 0x00d6,
  latinCapitalLetterUWithDiaeresis = 0x00dc,
  centSign = 0x00a2,
  poundSign = 0x00a3,
  yenSign = 0x00a5,
  pesetaSign = 0x20a7,
  latinSmallLetterFWithHook = 0x0192,

  // 160 - 175.
  latinSmallLetterAWithAcute = 0x00e1,
  latinSmallLetterIWithAcute = 0x00ed,
  latinSmallLetterOWithAcute = 0x00f3,
  latinSmallLetterUWithAcute = 0x00fa,
  latinSmallLetterNWithTilde = 0x00f1,
  latinCapitalLetterNWithTilde = 0x00d1,
  feminineOrdinalIndicator = 0x00aa,
  masculineOrdinalIndicator = 0x00ba,
  invertedQuestionMark = 0x00bf,
  reversedNotSign = 0x2310,
  notSign = 0x00ac,
  vulgarFractionOneHalf = 0x00bd,
  vulgarFractionOneQuarter = 0x00bc,
  invertedExclamationMark = 0x00a1,
  leftPointingDoubleAngleQuotationMark = 0x00ab,
  rightPointingDoubleAngleQuotationMark = 0x00bb,

  // 176 - 191.
  lightShade = 0x2591,
  mediumShade = 0x2592,
  darkShade = 0x2593,
  boxDrawingsLightVertical = 0x2502,
  boxDrawingsLightVerticalAndLeft = 0x2524,
  boxDrawingsVerticalSingleAndLeftDouble = 0x2561,
  boxDrawingsVerticalDoubleAndLeftSingle = 0x2562,
  boxDrawingsDownDoubleAndLeftSingle = 0x2556,
  boxDrawingsDownSingleAndLeftDouble = 0x2555,
  boxDrawingsDoubleVerticalAndLeft = 0x2563,
  boxDrawingsDoubleVertical = 0x2551,
  boxDrawingsDoubleDownAndLeft = 0x2557,
  boxDrawingsDoubleUpAndLeft = 0x255d,
  boxDrawingsUpDoubleAndLeftSingle = 0x255c,
  boxDrawingsUpSingleAndLeftDouble = 0x255b,
  boxDrawingsLightDownAndLeft = 0x2510,

  // 192 - 207.
  boxDrawingsLightUpAndRight = 0x2514,
  boxDrawingsLightUpAndHorizontal = 0x2534,
  boxDrawingsLightDownAndHorizontal = 0x252c,
  boxDrawingsLightVerticalAndRight = 0x251c,
  boxDrawingsLightHorizontal = 0x2500,
  boxDrawingsLightVerticalAndHorizontal = 0x253c,
  boxDrawingsVerticalSingleAndRightDouble = 0x255e,
  boxDrawingsVerticalDoubleAndRightSingle = 0x255f,
  boxDrawingsDoubleUpAndRight = 0x255a,
  boxDrawingsDoubleDownAndRight = 0x2554,
  boxDrawingsDoubleUpAndHorizontal = 0x2569,
  boxDrawingsDoubleDownAndHorizontal = 0x2566,
  boxDrawingsDoubleVerticalAndRight = 0x2560,
  boxDrawingsDoubleHorizontal = 0x2550,
  boxDrawingsDoubleVerticalAndHorizontal = 0x256c,
  boxDrawingsUpSingleAndHorizontalDouble = 0x2567,

  // 208 - 223.
  boxDrawingsUpDoubleAndHorizontalSingle = 0x2568,
  boxDrawingsDownSingleAndHorizontalDouble = 0x2564,
  boxDrawingsDownDoubleAndHorizontalSingle = 0x2565,
  boxDrawingsUpDoubleAndRightSingle = 0x2559,
  boxDrawingsUpSingleAndRightDouble = 0x2558,
  boxDrawingsDownSingleAndRightDouble = 0x2552,
  boxDrawingsDownDoubleAndRightSingle = 0x2553,
  boxDrawingsVerticalDoubleAndHorizontalSingle = 0x256b,
  boxDrawingsVerticalSingleAndHorizontalDouble = 0x256a,
  boxDrawingsLightUpAndLeft = 0x2518,
  boxDrawingsLightDownAndRight = 0x250c,
  fullBlock = 0x2588,
  lowerHalfBlock = 0x2584,
  leftHalfBlock = 0x258c,
  rightHalfBlock = 0x2590,
  upperHalfBlock = 0x2580,

  // 224 - 239.
  greekSmallLetterAlpha = 0x03b1,
  latinSmallLetterSharpS = 0x00df,
  greekCapitalLetterGamma = 0x0393,
  greekSmallLetterPi = 0x03c0,
  greekCapitalLetterSigma = 0x03a3,
  greekSmallLetterSigma = 0x03c3,
  microSign = 0x00b5,
  greekSmallLetterTau = 0x03c4,
  greekCapitalLetterPhi = 0x03a6,
  greekCapitalLetterTheta = 0x0398,
  greekCapitalLetterOmega = 0x03a9,
  greekSmallLetterDelta = 0x03b4,
  infinity = 0x221e,
  greekSmallLetterPhi = 0x03c6,
  greekSmallLetterEpsilon = 0x03b5,
  intersection = 0x2229,

  // 240 - 254.
  identicalTo = 0x2261,
  plusMinusSign = 0x00b1,
  greaterThanOrEqualTo = 0x2265,
  lessThanOrEqualTo = 0x2264,
  topHalfIntegral = 0x2320,
  bottomHalfIntegral = 0x2321,
  divisionSign = 0x00f7,
  almostEqualTo = 0x2248,
  degreeSign = 0x00b0,
  bulletOperator = 0x2219,
  middleDot = 0x00b7,
  squareRoot = 0x221a,
  superscriptLatinSmallLetterN = 0x207f,
  superscriptTwo = 0x00b2,
  blackSquare = 0x25a0
}
