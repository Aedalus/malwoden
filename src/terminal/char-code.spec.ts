import { CharCode } from "./char-code";

describe("CharCode", () => {
  it("Should have all CharCodes listed", () => {
    expect(CharCode).toMatchSnapshot();
  });
});
