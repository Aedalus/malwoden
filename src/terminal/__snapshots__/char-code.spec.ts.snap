// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharCode Should have all CharCodes listed 1`] = `
Object {
  "100": "dLower",
  "101": "eLower",
  "102": "fLower",
  "103": "gLower",
  "104": "hLower",
  "105": "iLower",
  "106": "jLower",
  "107": "kLower",
  "108": "lLower",
  "109": "mLower",
  "110": "nLower",
  "111": "oLower",
  "112": "pLower",
  "113": "qLower",
  "114": "rLower",
  "115": "sLower",
  "116": "tLower",
  "117": "uLower",
  "118": "vLower",
  "119": "wLower",
  "120": "xLower",
  "121": "yLower",
  "122": "zLower",
  "123": "leftBrace",
  "124": "pipe",
  "125": "rightBrace",
  "126": "tilde",
  "161": "invertedExclamationMark",
  "162": "centSign",
  "163": "poundSign",
  "165": "yenSign",
  "167": "sectionSign",
  "170": "feminineOrdinalIndicator",
  "171": "leftPointingDoubleAngleQuotationMark",
  "172": "notSign",
  "176": "degreeSign",
  "177": "plusMinusSign",
  "178": "superscriptTwo",
  "181": "microSign",
  "182": "pilcrow",
  "183": "middleDot",
  "186": "masculineOrdinalIndicator",
  "187": "rightPointingDoubleAngleQuotationMark",
  "188": "vulgarFractionOneQuarter",
  "189": "vulgarFractionOneHalf",
  "191": "invertedQuestionMark",
  "196": "latinCapitalLetterAWithDiaeresis",
  "197": "latinCapitalLetterAWithRingAbove",
  "198": "latinCapitalLetterAe",
  "199": "latinCapitalLetterCWithCedilla",
  "201": "latinCapitalLetterEWithAcute",
  "209": "latinCapitalLetterNWithTilde",
  "214": "latinCapitalLetterOWithDiaeresis",
  "220": "latinCapitalLetterUWithDiaeresis",
  "223": "latinSmallLetterSharpS",
  "224": "latinSmallLetterAWithGrave",
  "225": "latinSmallLetterAWithAcute",
  "226": "latinSmallLetterAWithCircumflex",
  "228": "latinSmallLetterAWithDiaeresis",
  "229": "latinSmallLetterAWithRingAbove",
  "230": "latinSmallLetterAe",
  "231": "latinSmallLetterCWithCedilla",
  "232": "latinSmallLetterEWithGrave",
  "233": "latinSmallLetterEWithAcute",
  "234": "latinSmallLetterEWithCircumflex",
  "235": "latinSmallLetterEWithDiaeresis",
  "236": "latinSmallLetterIWithGrave",
  "237": "latinSmallLetterIWithAcute",
  "238": "latinSmallLetterIWithCircumflex",
  "239": "latinSmallLetterIWithDiaeresis",
  "241": "latinSmallLetterNWithTilde",
  "242": "latinSmallLetterOWithGrave",
  "243": "latinSmallLetterOWithAcute",
  "244": "latinSmallLetterOWithCircumflex",
  "246": "latinSmallLetterOWithDiaeresis",
  "247": "divisionSign",
  "249": "latinSmallLetterUWithGrave",
  "250": "latinSmallLetterUWithAcute",
  "251": "latinSmallLetterUWithCircumflex",
  "252": "latinSmallLetterUWithDiaeresis",
  "255": "latinSmallLetterYWithDiaeresis",
  "32": "space",
  "33": "exclamationPoint",
  "34": "doubleQuote",
  "35": "numberSign",
  "36": "dollarSign",
  "37": "percent",
  "38": "ampersand",
  "39": "apostrophe",
  "40": "leftParenthesis",
  "402": "latinSmallLetterFWithHook",
  "41": "rightParenthesis",
  "42": "asterisk",
  "43": "plus",
  "44": "comma",
  "45": "minus",
  "46": "period",
  "47": "slash",
  "48": "zero",
  "49": "one",
  "50": "two",
  "51": "three",
  "52": "four",
  "53": "five",
  "54": "six",
  "55": "seven",
  "56": "eight",
  "57": "nine",
  "58": "colon",
  "59": "semicolon",
  "60": "lessThan",
  "61": "equals",
  "62": "greaterThan",
  "63": "questionMark",
  "64": "at",
  "65": "aUpper",
  "66": "bUpper",
  "67": "cUpper",
  "68": "dUpper",
  "69": "eUpper",
  "70": "fUpper",
  "71": "gUpper",
  "72": "hUpper",
  "73": "iUpper",
  "74": "jUpper",
  "75": "kUpper",
  "76": "lUpper",
  "77": "mUpper",
  "78": "nUpper",
  "79": "oUpper",
  "80": "pUpper",
  "81": "qUpper",
  "82": "rUpper",
  "8226": "bullet",
  "8252": "doubleExclamationMark",
  "83": "sUpper",
  "8319": "superscriptLatinSmallLetterN",
  "8359": "pesetaSign",
  "84": "tUpper",
  "85": "uUpper",
  "8592": "leftwardsArrow",
  "8593": "upwardsArrow",
  "8594": "rightwardsArrow",
  "8595": "downwardsArrow",
  "8596": "leftRightArrow",
  "8597": "upDownArrow",
  "86": "vUpper",
  "8616": "upDownArrowWithBase",
  "87": "wUpper",
  "8729": "bulletOperator",
  "8730": "squareRoot",
  "8734": "infinity",
  "8735": "rightAngle",
  "8745": "intersection",
  "8776": "almostEqualTo",
  "88": "xUpper",
  "8801": "identicalTo",
  "8804": "lessThanOrEqualTo",
  "8805": "greaterThanOrEqualTo",
  "89": "yUpper",
  "8962": "house",
  "8976": "reversedNotSign",
  "8992": "topHalfIntegral",
  "8993": "bottomHalfIntegral",
  "90": "zUpper",
  "91": "leftBracket",
  "915": "greekCapitalLetterGamma",
  "92": "backSlash",
  "920": "greekCapitalLetterTheta",
  "93": "rightBracket",
  "931": "greekCapitalLetterSigma",
  "934": "greekCapitalLetterPhi",
  "937": "greekCapitalLetterOmega",
  "94": "caret",
  "945": "greekSmallLetterAlpha",
  "9472": "boxDrawingsLightHorizontal",
  "9474": "boxDrawingsLightVertical",
  "948": "greekSmallLetterDelta",
  "9484": "boxDrawingsLightDownAndRight",
  "9488": "boxDrawingsLightDownAndLeft",
  "949": "greekSmallLetterEpsilon",
  "9492": "boxDrawingsLightUpAndRight",
  "9496": "boxDrawingsLightUpAndLeft",
  "95": "underscore",
  "9500": "boxDrawingsLightVerticalAndRight",
  "9508": "boxDrawingsLightVerticalAndLeft",
  "9516": "boxDrawingsLightDownAndHorizontal",
  "9524": "boxDrawingsLightUpAndHorizontal",
  "9532": "boxDrawingsLightVerticalAndHorizontal",
  "9552": "boxDrawingsDoubleHorizontal",
  "9553": "boxDrawingsDoubleVertical",
  "9554": "boxDrawingsDownSingleAndRightDouble",
  "9555": "boxDrawingsDownDoubleAndRightSingle",
  "9556": "boxDrawingsDoubleDownAndRight",
  "9557": "boxDrawingsDownSingleAndLeftDouble",
  "9558": "boxDrawingsDownDoubleAndLeftSingle",
  "9559": "boxDrawingsDoubleDownAndLeft",
  "9560": "boxDrawingsUpSingleAndRightDouble",
  "9561": "boxDrawingsUpDoubleAndRightSingle",
  "9562": "boxDrawingsDoubleUpAndRight",
  "9563": "boxDrawingsUpSingleAndLeftDouble",
  "9564": "boxDrawingsUpDoubleAndLeftSingle",
  "9565": "boxDrawingsDoubleUpAndLeft",
  "9566": "boxDrawingsVerticalSingleAndRightDouble",
  "9567": "boxDrawingsVerticalDoubleAndRightSingle",
  "9568": "boxDrawingsDoubleVerticalAndRight",
  "9569": "boxDrawingsVerticalSingleAndLeftDouble",
  "9570": "boxDrawingsVerticalDoubleAndLeftSingle",
  "9571": "boxDrawingsDoubleVerticalAndLeft",
  "9572": "boxDrawingsDownSingleAndHorizontalDouble",
  "9573": "boxDrawingsDownDoubleAndHorizontalSingle",
  "9574": "boxDrawingsDoubleDownAndHorizontal",
  "9575": "boxDrawingsUpSingleAndHorizontalDouble",
  "9576": "boxDrawingsUpDoubleAndHorizontalSingle",
  "9577": "boxDrawingsDoubleUpAndHorizontal",
  "9578": "boxDrawingsVerticalSingleAndHorizontalDouble",
  "9579": "boxDrawingsVerticalDoubleAndHorizontalSingle",
  "9580": "boxDrawingsDoubleVerticalAndHorizontal",
  "96": "accent",
  "960": "greekSmallLetterPi",
  "9600": "upperHalfBlock",
  "9604": "lowerHalfBlock",
  "9608": "fullBlock",
  "9612": "leftHalfBlock",
  "9616": "rightHalfBlock",
  "9617": "lightShade",
  "9618": "mediumShade",
  "9619": "darkShade",
  "963": "greekSmallLetterSigma",
  "9632": "blackSquare",
  "964": "greekSmallLetterTau",
  "9644": "blackRectangle",
  "9650": "blackUpPointingTriangle",
  "9658": "blackRightPointingPointer",
  "966": "greekSmallLetterPhi",
  "9660": "blackDownPointingTriangle",
  "9668": "blackLeftPointingPointer",
  "9675": "whiteCircle",
  "9688": "inverseBullet",
  "9689": "inverseWhiteCircle",
  "97": "aLower",
  "9786": "whiteSmilingFace",
  "9787": "blackSmilingFace",
  "9788": "whiteSunWithRays",
  "9792": "femaleSign",
  "9794": "maleSign",
  "98": "bLower",
  "9824": "blackSpadeSuit",
  "9827": "blackClubSuit",
  "9829": "blackHeartSuit",
  "9830": "blackDiamondSuit",
  "9834": "eighthNote",
  "9835": "beamedEighthNotes",
  "99": "cLower",
  "aLower": 97,
  "aUpper": 65,
  "accent": 96,
  "almostEqualTo": 8776,
  "ampersand": 38,
  "apostrophe": 39,
  "asterisk": 42,
  "at": 64,
  "bLower": 98,
  "bUpper": 66,
  "backSlash": 92,
  "beamedEighthNotes": 9835,
  "blackClubSuit": 9827,
  "blackDiamondSuit": 9830,
  "blackDownPointingTriangle": 9660,
  "blackHeartSuit": 9829,
  "blackLeftPointingPointer": 9668,
  "blackRectangle": 9644,
  "blackRightPointingPointer": 9658,
  "blackSmilingFace": 9787,
  "blackSpadeSuit": 9824,
  "blackSquare": 9632,
  "blackUpPointingTriangle": 9650,
  "bottomHalfIntegral": 8993,
  "boxDrawingsDoubleDownAndHorizontal": 9574,
  "boxDrawingsDoubleDownAndLeft": 9559,
  "boxDrawingsDoubleDownAndRight": 9556,
  "boxDrawingsDoubleHorizontal": 9552,
  "boxDrawingsDoubleUpAndHorizontal": 9577,
  "boxDrawingsDoubleUpAndLeft": 9565,
  "boxDrawingsDoubleUpAndRight": 9562,
  "boxDrawingsDoubleVertical": 9553,
  "boxDrawingsDoubleVerticalAndHorizontal": 9580,
  "boxDrawingsDoubleVerticalAndLeft": 9571,
  "boxDrawingsDoubleVerticalAndRight": 9568,
  "boxDrawingsDownDoubleAndHorizontalSingle": 9573,
  "boxDrawingsDownDoubleAndLeftSingle": 9558,
  "boxDrawingsDownDoubleAndRightSingle": 9555,
  "boxDrawingsDownSingleAndHorizontalDouble": 9572,
  "boxDrawingsDownSingleAndLeftDouble": 9557,
  "boxDrawingsDownSingleAndRightDouble": 9554,
  "boxDrawingsLightDownAndHorizontal": 9516,
  "boxDrawingsLightDownAndLeft": 9488,
  "boxDrawingsLightDownAndRight": 9484,
  "boxDrawingsLightHorizontal": 9472,
  "boxDrawingsLightUpAndHorizontal": 9524,
  "boxDrawingsLightUpAndLeft": 9496,
  "boxDrawingsLightUpAndRight": 9492,
  "boxDrawingsLightVertical": 9474,
  "boxDrawingsLightVerticalAndHorizontal": 9532,
  "boxDrawingsLightVerticalAndLeft": 9508,
  "boxDrawingsLightVerticalAndRight": 9500,
  "boxDrawingsUpDoubleAndHorizontalSingle": 9576,
  "boxDrawingsUpDoubleAndLeftSingle": 9564,
  "boxDrawingsUpDoubleAndRightSingle": 9561,
  "boxDrawingsUpSingleAndHorizontalDouble": 9575,
  "boxDrawingsUpSingleAndLeftDouble": 9563,
  "boxDrawingsUpSingleAndRightDouble": 9560,
  "boxDrawingsVerticalDoubleAndHorizontalSingle": 9579,
  "boxDrawingsVerticalDoubleAndLeftSingle": 9570,
  "boxDrawingsVerticalDoubleAndRightSingle": 9567,
  "boxDrawingsVerticalSingleAndHorizontalDouble": 9578,
  "boxDrawingsVerticalSingleAndLeftDouble": 9569,
  "boxDrawingsVerticalSingleAndRightDouble": 9566,
  "bullet": 8226,
  "bulletOperator": 8729,
  "cLower": 99,
  "cUpper": 67,
  "caret": 94,
  "centSign": 162,
  "colon": 58,
  "comma": 44,
  "dLower": 100,
  "dUpper": 68,
  "darkShade": 9619,
  "degreeSign": 176,
  "divisionSign": 247,
  "dollarSign": 36,
  "doubleExclamationMark": 8252,
  "doubleQuote": 34,
  "downwardsArrow": 8595,
  "eLower": 101,
  "eUpper": 69,
  "eight": 56,
  "eighthNote": 9834,
  "equals": 61,
  "exclamationPoint": 33,
  "fLower": 102,
  "fUpper": 70,
  "femaleSign": 9792,
  "feminineOrdinalIndicator": 170,
  "five": 53,
  "four": 52,
  "fullBlock": 9608,
  "gLower": 103,
  "gUpper": 71,
  "greaterThan": 62,
  "greaterThanOrEqualTo": 8805,
  "greekCapitalLetterGamma": 915,
  "greekCapitalLetterOmega": 937,
  "greekCapitalLetterPhi": 934,
  "greekCapitalLetterSigma": 931,
  "greekCapitalLetterTheta": 920,
  "greekSmallLetterAlpha": 945,
  "greekSmallLetterDelta": 948,
  "greekSmallLetterEpsilon": 949,
  "greekSmallLetterPhi": 966,
  "greekSmallLetterPi": 960,
  "greekSmallLetterSigma": 963,
  "greekSmallLetterTau": 964,
  "hLower": 104,
  "hUpper": 72,
  "house": 8962,
  "iLower": 105,
  "iUpper": 73,
  "identicalTo": 8801,
  "infinity": 8734,
  "intersection": 8745,
  "inverseBullet": 9688,
  "inverseWhiteCircle": 9689,
  "invertedExclamationMark": 161,
  "invertedQuestionMark": 191,
  "jLower": 106,
  "jUpper": 74,
  "kLower": 107,
  "kUpper": 75,
  "lLower": 108,
  "lUpper": 76,
  "latinCapitalLetterAWithDiaeresis": 196,
  "latinCapitalLetterAWithRingAbove": 197,
  "latinCapitalLetterAe": 198,
  "latinCapitalLetterCWithCedilla": 199,
  "latinCapitalLetterEWithAcute": 201,
  "latinCapitalLetterNWithTilde": 209,
  "latinCapitalLetterOWithDiaeresis": 214,
  "latinCapitalLetterUWithDiaeresis": 220,
  "latinSmallLetterAWithAcute": 225,
  "latinSmallLetterAWithCircumflex": 226,
  "latinSmallLetterAWithDiaeresis": 228,
  "latinSmallLetterAWithGrave": 224,
  "latinSmallLetterAWithRingAbove": 229,
  "latinSmallLetterAe": 230,
  "latinSmallLetterCWithCedilla": 231,
  "latinSmallLetterEWithAcute": 233,
  "latinSmallLetterEWithCircumflex": 234,
  "latinSmallLetterEWithDiaeresis": 235,
  "latinSmallLetterEWithGrave": 232,
  "latinSmallLetterFWithHook": 402,
  "latinSmallLetterIWithAcute": 237,
  "latinSmallLetterIWithCircumflex": 238,
  "latinSmallLetterIWithDiaeresis": 239,
  "latinSmallLetterIWithGrave": 236,
  "latinSmallLetterNWithTilde": 241,
  "latinSmallLetterOWithAcute": 243,
  "latinSmallLetterOWithCircumflex": 244,
  "latinSmallLetterOWithDiaeresis": 246,
  "latinSmallLetterOWithGrave": 242,
  "latinSmallLetterSharpS": 223,
  "latinSmallLetterUWithAcute": 250,
  "latinSmallLetterUWithCircumflex": 251,
  "latinSmallLetterUWithDiaeresis": 252,
  "latinSmallLetterUWithGrave": 249,
  "latinSmallLetterYWithDiaeresis": 255,
  "leftBrace": 123,
  "leftBracket": 91,
  "leftHalfBlock": 9612,
  "leftParenthesis": 40,
  "leftPointingDoubleAngleQuotationMark": 171,
  "leftRightArrow": 8596,
  "leftwardsArrow": 8592,
  "lessThan": 60,
  "lessThanOrEqualTo": 8804,
  "lightShade": 9617,
  "lowerHalfBlock": 9604,
  "mLower": 109,
  "mUpper": 77,
  "maleSign": 9794,
  "masculineOrdinalIndicator": 186,
  "mediumShade": 9618,
  "microSign": 181,
  "middleDot": 183,
  "minus": 45,
  "nLower": 110,
  "nUpper": 78,
  "nine": 57,
  "notSign": 172,
  "numberSign": 35,
  "oLower": 111,
  "oUpper": 79,
  "one": 49,
  "pLower": 112,
  "pUpper": 80,
  "percent": 37,
  "period": 46,
  "pesetaSign": 8359,
  "pilcrow": 182,
  "pipe": 124,
  "plus": 43,
  "plusMinusSign": 177,
  "poundSign": 163,
  "qLower": 113,
  "qUpper": 81,
  "questionMark": 63,
  "rLower": 114,
  "rUpper": 82,
  "reversedNotSign": 8976,
  "rightAngle": 8735,
  "rightBrace": 125,
  "rightBracket": 93,
  "rightHalfBlock": 9616,
  "rightParenthesis": 41,
  "rightPointingDoubleAngleQuotationMark": 187,
  "rightwardsArrow": 8594,
  "sLower": 115,
  "sUpper": 83,
  "sectionSign": 167,
  "semicolon": 59,
  "seven": 55,
  "six": 54,
  "slash": 47,
  "space": 32,
  "squareRoot": 8730,
  "superscriptLatinSmallLetterN": 8319,
  "superscriptTwo": 178,
  "tLower": 116,
  "tUpper": 84,
  "three": 51,
  "tilde": 126,
  "topHalfIntegral": 8992,
  "two": 50,
  "uLower": 117,
  "uUpper": 85,
  "underscore": 95,
  "upDownArrow": 8597,
  "upDownArrowWithBase": 8616,
  "upperHalfBlock": 9600,
  "upwardsArrow": 8593,
  "vLower": 118,
  "vUpper": 86,
  "vulgarFractionOneHalf": 189,
  "vulgarFractionOneQuarter": 188,
  "wLower": 119,
  "wUpper": 87,
  "whiteCircle": 9675,
  "whiteSmilingFace": 9786,
  "whiteSunWithRays": 9788,
  "xLower": 120,
  "xUpper": 88,
  "yLower": 121,
  "yUpper": 89,
  "yenSign": 165,
  "zLower": 122,
  "zUpper": 90,
  "zero": 48,
}
`;
